interface/usergroup/user_admin.php

```diff
@@ -559,7 +565,56 @@ function authorized_clicked() {
   </select></td>
   <td><span class=text><?php echo xlt('Additional Info'); ?>:</span></td>
   <td><textarea style="width:150px;" name="comments" wrap=auto rows=4 cols=25><?php echo $iter["info"];?></textarea></td>
+  </tr>
+  <tr>
+  <td><span class="text"><?php echo xlt('Full screen role'); ?>:</span></td>
+  <td>
+  <select style="width:120px;" name="role_name">
+    <?php
+      $fres = sqlStatement("select distinct menu_set from menu_trees order by menu_set");
```

Comment:
I suppose we don't really need binding here due to no input variables?

Reply:
Can't seem to comment on some of your comments for some reason
$query = "UPDATE users SET fullscreen_role = '".$tqver."' WHERE id =...... right?.
I was following the other query statements in the same file. I can change it to concatenation if that is needed.
Yes, no binding is needed in this one.